django/website/main/templates/base.html

```diff
@@ -23,20 +23,18 @@ <h1 class="logo"><a href="/">{% settings_value "SITE_NAME" %}</a></h1>
         {% block logframe_switching_form %}
         {% if request.user and request.user.is_authenticated %}
         <form id="logframes-switch" action="{% url "switch-logframes" %}" method="post">{% csrf_token %}
-            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();"{% if logframe_list|length == 1 %}disabled="disabled"{% endif %}>
+            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();">
             	{% for logframe in logframe_list %}<option value="{{ logframe.id }}"{% if request.user.preferences.last_viewed_logframe == logframe %} selected="selected"{% endif %}>{{ logframe.name }}</option>{% endfor %}
+            	<option value="-1">{% trans "+ Create a new logframe" %}</option>
```

Comment:
The selector allows me to navigate between the different logframes or create a new one. It also shows me which one I'm viewing currently. So I would expect it to show me the name of the log frame I'm currently viewing or "Create a new Logframe" if I'm doing that.

I would value Ian's input on this